src/ResourceManager/AzureBatch/Commands.Batch/Commands.Batch.csproj

```diff
@@ -61,6 +61,12 @@
       <HintPath>..\..\..\packages\WindowsAzure.Storage.6.2.0\lib\net40\Microsoft.WindowsAzure.Storage.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
@antoniowmsft please remove the references to `Newtonsoft.Json` and `System.Management.Automation` added in this file as they are already referenced in a separate common targets file

Reply:
Done.